packages/executor/src/executor.ts

```diff
@@ -157,28 +157,11 @@ export class Executor {
       this.bundlingService.setBundlingMode("manual");
       this.logger.info(`${this.networkName}: [X] MANUAL BUNDLING`);
     }
+
     if (this.config.testingMode) {
       this.bundlingService.setMaxBundleSize(10);
     }
 
```

Comment:
Why has this block been removed?

Reply:
I moved those checks to their corresponding relayers in `services/BundlingService/relayers` folder